migration/iD11/oracle/202309291327_IDEMPIERE-5836.sql

```diff
@@ -0,0 +1,16 @@
+SELECT register_migration_script('202309291327_IDEMPIERE-5836.sql') FROM dual;
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+-- Aug 8, 2023, 1:27:56 PM IST
+UPDATE AD_Field SET DisplayLogic='@IsSummary@=''N'' & @ProductType@=I | @ProductType@=E', AD_Reference_Value_ID=NULL, AD_Val_Rule_ID=NULL, IsToolbarButton=NULL,Updated=TO_DATE('2023-08-08 13:27:56','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=1026
```

Comment:
The logic here is wrong, is showing the Stocked flag when the product is marked as Summary and Expense.

You need to add parenthesis on the OR condition, or reorder it to check the IsSummary at the end